packages/mobile/src/screens/auth/auth.tsx

```diff
@@ -26,22 +35,31 @@ import {
 import { styles } from './styles';
 
 const Auth: FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
   const { name } = useRoute();
+  const dataStatus = useAppSelector(selectUserActionDataStatus);
   const dispatch = useAppDispatch();
   const { colors } = useCustomTheme();
   const { t } = useTranslation();
-  const navigation = useNavigation();
+  const navigation: NavigationProp<MainNavigationParamList> = useNavigation();
   const screenLabel =
     name === RootScreenName.SIGN_IN
       ? t('verification.SING_IN')
       : t('verification.CREATE_ACCOUNT');
 
   const handleSignIn = (payload: UserSignInDto): void => {
+    setIsLoading(true);
     dispatch(authActions.signIn(payload));
+    setIsLoading(false);
```

Comment:
As I know it won't work as it expected to, because ```dispatch``` doesn't work as f.e. ```await``` so it won't wait until ```dispatch``` is done. @AleksandrTermenzhy correct if I am wrong

Reply:
changed to data-status checking for spinner